### PR TITLE
Added three seed projects for new team leader.

### PR DESCRIPTION
**Pull request by jordanh**
Implements #632

### CODE REVIEW

**Comment by ackernaut** [comment]
Oh snap!

File: src/server/graphql/models/Team/helpers/addSeedProjects.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    isArchived: false,
+    createdAt: now,
+    createdBy: userId,
+    teamId,
+    teamMemberId: `${userId}::${teamId}`,
+    userId,
+    updatedAt: now
+  }));
+
+  return await r.table('Project').insert(seedProjects)
```
Comment (mattkrick) [inline src/server/graphql/models/Team/helpers/addSeedProjects.js]: does this work without the `returnChanges: true` option?

**Comment by jordanh** [comment]
Whoa @mattkrick good catch on the `returnChanges`. You know, I tested it and it _did_ work and _still_ worked without `returnChanges`. Makes me wonder what the defaults of our driver are...cause the rethinkdb doc defaults definitely have `returnChanges: false`.

**Comment by jordanh** [comment]
GTG now?

**Pull request merged**